kuma/users/models.py

```diff
@@ -194,3 +194,7 @@ def wiki_revisions(self, count=5):
 
     def allows_editing_by(self, user):
         return user.is_staff or user.is_superuser or user.pk == self.pk
+
+    def has_akismet_submission_permission(self):
+        return (self.groups.filter(permissions__codename=u'add_revisionakismetsubmission').exists() or
+                 self.user_permissions.filter(codename=u'add_revisionakismetsubmission').exists())
```

Comment:
Please use this instead: ``has_perm("wiki.add_revisionakismetsubmission")``, which does the equivalent and more.